src/index.js

```diff
@@ -3,8 +3,15 @@ import ReactDOM from 'react-dom/client';
 import './index.css';
 import { Provider } from 'react-redux';
 import { BrowserRouter as Router } from 'react-router-dom';
-import App from './App';
+import App from './components/App/App';
 import store from './redux/configureStore';
+import { login } from './redux/auth';
+
+const user = {
+  name: 'Solomon',
+};
+
+store.dispatch(login(user));
```

Comment:
I believe the above piece of code was used for testing so I suggest you remove them to keep the code clean.